### ISSUE TITLE
heroku pipelines:promote exit code

### ISSUE THREAD

**Issue by onesupercoder** [heroku/heroku-pipelines#66]
I was wondering if we could get the heroku pipelines:promote to give a failing exit code on failure instead of a success.

**Comment by gudmundur** [comment]
@onesupercoder That totally makes sense. I'll bring it up with the team and try to get a PR open for this in the coming days.

**Comment by onesupercoder** [comment]
Thanks a million

**Comment by onesupercoder** [comment]
Yeah i have no idea if it is something as easy as, https://github.com/heroku/heroku-pipelines/commit/a2e889bc2d76209e2985650cb815f9b31eb18f2d or something way more complicated.
